File: web2projekat/src/models/UserDto.js
export default class UserDto {
    constructor(data) {
        this.Id = data.id;
        this.Username = data.username;
        this.Password = data.password;
        this.Email = data.email;
        this.FullName = data.fullName;
        this.DateOfBirth = data.dateOfBirth;
        this.Address = data.address;
        this.UserImage = data.userImage;
        this.UserType = data.userType;
        this.Verified = data.verified;
        this.VerificationStatus = data.verificationStatus
    }
}

File: WebServer/WebServer/Dto/UserDto.cs
using System.ComponentModel.DataAnnotations;

namespace WebServer.Dto
{
    public class UserDto
    {
        public long Id { get; set; }

        [Required]
        public string Username { get; set; }

        [Required]
        public string Password { get; set; }

        [Required]
        public string Email { get; set; }

        [Required]
        public string FullName { get; set; }

        [Required]
        public DateTime DateOfBirth { get; set; }

        [Required]
        public string Address { get; set; }

        public string UserImage { get; set; }

        [Required]
        public string UserType { get; set; }

        public string VerificationStatus { get; set; }

        public bool Verified { get; set; }

        //public double DeliveryPrice { get; set; }
    }
}
